apps/admin_app/assets/js/views/order/index.js

```diff
@@ -4,13 +4,13 @@ export default class View extends MainView {
     mount() {
       super.mount();
       console.log('order view mounted');
+      let url = "/orders/pending";
```

Comment:
I don't understand the purpose of the link, check the active link and you can use that to make the request.